scripts/minikube-minishift-setup-env.sh

```diff
@@ -62,13 +62,23 @@ case ${1} in
         fi
         
         minikube version
+        # Setup to find nessasary data from cluster setup
+        ## Constants
+        SETUP_OPERATORS="./scripts/configure-cluster/common/setup-operators.sh"
+
         set +x
         # Get kubectl cluster info
         kubectl cluster-info
 
+        # Enable OLM for running operator tests
+        curl -sL https://github.com/operator-framework/operator-lifecycle-manager/releases/download/v0.17.0/install.sh | bash -s v0.17.0
```

Comment:
Make the version a variable right before this line. OLM [v0.18.2 is already available in upstream](https://github.com/operator-framework/operator-lifecycle-manager/releases/tag/v0.18.2) and I expect this version thingy might change in future for our repo as more updates come to SBO.

Reply:
done